cdap-ui/app/cdap/components/PreviewData/Table.tsx

```diff
@@ -121,7 +130,9 @@ const DataTableView: React.FC<IDataTableProps> = ({
               <TableRow className={classes.row} key={`tr-${j}`}>
                 {headers.map((fieldName, k) => {
                   return (
-                    <CustomTableCell key={`table-cell-${k}`}>{record[fieldName]}</CustomTableCell>
+                    <CustomTableCell key={`table-cell-${k}`}>
+                      {format(record[fieldName])}
```

Comment:
Do we need to format the headers? Aren't these just column names?

Reply:
I thought there was a small chance the column names could also be objects, in the case of nested schemas.

Reply:
Hhmm I am not sure if thats how nested schema column names would be. It will most likely be a single column whose values could be tables. We can leave this as is here until we manually verify nested schemas.

Reply:
I will add a comment for context.